public/game.html

```diff
@@ -0,0 +1,60 @@
+<html>
+  <head>
+    <title>タイピングゲーム</title>
+    <link rel="stylesheet" href="/stylesheets/game.css" />
+    <script type="module" src="/javascripts/main.js"></script>
+  </head>
+
+  <body>
+    <div data-screen="game">
+      <div class="game-header">
+        <div class="point">100,000pt</div>
+        <div class="timer">0:00</div>
+      </div>
+      <div class="game-body">
+        <div class="typing">
+          <div class="arrow_box">
+            <br />
+            <div class="textline">
+              <p class="word">ディレクトリトラバーサル</p>
+            </div>
+            <div class="textline">
+              <span class="red">D</span>
+              <span class="red">E</span>
+              <span class="red">L</span>
+              <span class="red">I</span>
+              <span class="red">R</span>
+              <span class="red">E</span>
+              <span class="red">K</span>
+              <span class="red">U</span>
+              <span class="red">T</span>
+              <span class="red">O</span>
+              <span class="">R</span>
+              <span class="">I</span>
+              <span class="">T</span>
+              <span class="">O</span>
+              <span class="">R</span>
+              <span class="">I</span>
+              <span class="">T</span>
+              <span class="">O</span>
+              <span class="">R</span>
+              <span class="">A</span>
+              <span class="">B</span>
+              <span class="">A</span>
+              <span class="">-</span>
+              <span class="">S</span>
+              <span class="">A</span>
+              <span class="">R</span>
+              <span class="">U</span>
+            </div>
+            <br />
+          </div>
+          <br /><br /><br />
```

Comment:
`.array_box`と同じ階層に空のdivを書く
```suggestion
<div></div>
```